functions/Get-DbaAgentJob.ps1

```diff
@@ -67,18 +87,22 @@ FUNCTION Get-DbaAgentJob {
 			}
 
 			$jobs = $server.JobServer.Jobs
+			
 			if ($Job) {
 				$jobs = $jobs | Where-Object Name -In $Job
 			}
 			if ($ExcludeJob) {
 				$jobs = $jobs | Where-Object Name -NotIn $ExcludeJob
 			}
-
+			if (!$IncludeDisabled -and !$job) {
```

Comment:
as far as we figure out a logic and every cmdlet tries to follow it, it's good (i.e. -nooranges or -apples are both logic)